docs(registration): DOC-445 add manual id steps

317brian opened the pull request:
Resolves Jira: [DOC-445]


[DOC-445]: https://armory.atlassian.net/browse/DOC-445

317brian committed:
docs(registration): add manual id steps

317brian committed:
DOC-445 fix typo

aimeeu commented:
deploy preview https://deploy-preview-796--armory-docs.netlify.app/docs/installation/deployment-reg/

aimeeu committed:
copy edits

fixed indent in list under Register your environment bullet 2
fixed broken links due to section name change

aimeeu committed:
try to fix note

aimeeu committed:
change note to bold

note won't render in bullet list

KathrynLewis commented on content/en/docs/installation/deployment-reg.md:
```diff
@@ -12,23 +12,25 @@ toc_hide: true
   - `spec.spinnakerConfig.profiles.spinnaker.”armory.cloud”.iam.tokenIssueUrl`: set this value to `https://auth.cloud.armory.io/oauth/token`.
-  - `spec.spinnakerConfig.profiles.spinnaker.”armory.cloud”.iam.clientID`: set this value to **Client ID** that you received from [Get your registration information](#get-your-registration-information)
-  - `spec.spinnakerConfig.profiles.spinnaker.”armory.cloud”.iam.clientSecret`: set this value to the **Secret** that you received from [Get your registration information](#get-your-registration-information).
+  - `spec.spinnakerConfig.profiles.spinnaker.”armory.cloud”.iam.clientID`: set this value to **Client ID** that you received from [Register your environment](#register-your-environment).
```
AFAIK, the armory.cloud bit doesn't need to be in quotes. The dots are stand-ins for indentation in yaml. 

This:
```
armory.cloud:
  iam: ...
```
is the same as this:
```
armory:
  cloud:
    iam: ...
```
Unless of course you have some other reason for putting it in quotes ¯\\_(ツ)_/¯

317brian replied:
🕵🏼‍♀️

317brian committed:
remove unnecessary quotes
